packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-line.tsx

```diff
@@ -148,10 +148,10 @@ const Line = (props: LineProps) => {
     if (extend) {
         const trimmedRange = trimRange(range, graphDimensionsInPixels);
         startExtend = extend.start
-            ? getIntersectionOfRayWithBox(start, end, trimmedRange)
+            ? getIntersectionOfRayWithBox(end, start, trimmedRange)
```

Comment:
Not directly related to this PR, but Nisha recently changed the locked line component to simply use `point1` and `point2` for lines. This feels like it might be applicable for the `MovableLine` component(s) as well as for line and ray they don't really have a start/end but rather two points that the line/ray intersect.  🤔  

For a future PR if you agree.